backend/src/graphql/resolver/AdminResolver.ts

```diff
@@ -239,59 +244,63 @@ export class AdminResolver {
 
     const moderator = getUser(context)
 
-    const pendingCreationToUpdate = await AdminPendingCreation.findOne({ id })
+    const contributionToUpdate = await Contribution.findOne({ id })
 
-    if (!pendingCreationToUpdate) {
-      throw new Error('No creation found to given id.')
+    if (!contributionToUpdate) {
+      throw new Error('No contribution found to given id.')
     }
 
-    if (pendingCreationToUpdate.userId !== user.id) {
-      throw new Error('user of the pending creation and send user does not correspond')
+    if (contributionToUpdate.userId !== user.id) {
+      throw new Error('user of the pending contribution and send user does not correspond')
     }
 
     const creationDateObj = new Date(creationDate)
     let creations = await getUserCreation(user.id)
-    if (pendingCreationToUpdate.date.getMonth() === creationDateObj.getMonth()) {
-      creations = updateCreations(creations, pendingCreationToUpdate)
+    if (contributionToUpdate.contributionDate.getMonth() === creationDateObj.getMonth()) {
+      creations = updateCreations(creations, contributionToUpdate)
```

Comment:
shouldn't the method be renamed to updateContributions()

Reply:
In this case, `creations` are the available amounts for the last three months